src/components/tl/mlx5/tl_mlx5.h

```diff
@@ -140,7 +137,11 @@ typedef struct ucc_tl_mlx5_team {
     ucc_tl_mlx5_alltoall_t         *a2a;
     ucc_topo_t                     *topo;
     ucc_ep_map_t                    ctx_map;
+    int                             local_mcast_ctx_ready;
     ucc_tl_mlx5_mcast_team_t       *mcast;
+    ucc_status_t                   *local_status_array;
+    ucc_status_t                   *global_status_array;
+    int                             global_sync_in_progress;
```

Comment:
in my opinion this bool flag is not necessary, the check can be done with `tl_team->scoll_req`

Reply:
I agree, but this is just more readable and easier to understand. Especially that the scoll_req variable is used for many scenarios, not just for team create global status check.